runtime/src/main/java/org/corfudb/runtime/clients/BaseClient.java

```diff
@@ -57,9 +57,9 @@ public boolean pingSync() {
      * @param newEpoch New Epoch to be set
      * @return Completable future which returns true on successful epoch set.
      */
-    public CompletableFuture<Boolean> setRemoteEpoch(long newEpoch) {
-        CorfuMsg msg = new CorfuPayloadMsg<>(CorfuMsgType.SET_EPOCH, newEpoch).setEpoch(epoch);
-        log.info("setRemoteEpoch: send SET_EPOCH from me(clientId={}) to new epoch {}",
+    public CompletableFuture<Boolean> sealRemote(long newEpoch) {
```

Comment:
Maybe sealServer or sealRemoteServer is more appropriate?

Reply:
Done.